Neo4jClient/packages.config

```diff
@@ -2,4 +2,5 @@
 <packages>
   <package id="Microsoft.Net.Http" version="2.0.20710.0" targetFramework="net40" />
   <package id="Newtonsoft.Json" version="6.0.3" targetFramework="net40" />
+  <package id="TaskHelpers.Sources" version="0.3" targetFramework="net40" />

```

Comment:
I figured you would still want to keep .NET 4 support. So, I kept working with pure TPL APIs without `async` and `await`. However, I pulled in this package as it makes it really easy to work with TPL APIs safely. This was originally used in ASP.NET Web API project when it was supporting .NET 4.0. I spot some places where these extension methods can help to make them more safe and less error free but that would make a different PR.

Hope it's OK to have this package.